src/renderer/page/file/view.jsx

```diff
@@ -163,90 +159,85 @@ class FilePage extends React.Component<Props> {
 
     return (
       <Page forContent>
-        <section className="content__wrapper">
-          {showFile && <FileViewer className="content__embedded" uri={uri} mediaType={mediaType} />}
-          {!showFile &&
-            (thumbnail ? (
-              <Thumbnail shouldObscure={shouldObscureThumbnail} src={thumbnail} />
-            ) : (
-              <div
-                className={classnames('content__empty', {
-                  'content__empty--nsfw': shouldObscureThumbnail,
-                })}
-              >
-                <div className="card__media-text">
-                  {__("Sorry, looks like we can't preview this file.")}
-                </div>
+        {showFile && <FileViewer className="content__embedded" uri={uri} mediaType={mediaType} />}
```

Comment:
If we are removing the wrapper on this, we should probably add some sort of max-height, then add a background so the thumbnail, content doesn't get huge. Most thumbails aren't great on large screens, and this will only be exacerbated on larger screens. 
 
<img width="400" alt="screen shot 2018-12-12 at 3 28 55 pm" src="https://user-images.githubusercontent.com/16882830/49896899-a9728d00-fe22-11e8-88f0-ee6404a1040f.png">

<img width="400" alt="screen shot 2018-12-12 at 3 30 24 pm" src="https://user-images.githubusercontent.com/16882830/49897005-e8a0de00-fe22-11e8-85d1-3ff556a355b9.png">